appscale/tools/node_layout.py

```diff
@@ -513,6 +513,19 @@ def db_master(self):
         return node
     return None
 
+  def are_disks_used(self):
+    """ Searches through the nodes in this NodeLayout to see if any persistent
+    disks are being used.
+
+    Returns:
+      True if any persistent disks are used, and False otherwise.
+    """
+    disks = [node.disk for node in self.nodes]
```

Comment:
This is perfectly acceptable, but I'd just like to share the convenience of `any`:
`return any(node.disk for node in self.nodes)`